bundles/net.tourbook/src/net/tourbook/tag/Dialog_TourTag.java

```diff
@@ -124,24 +159,68 @@ private void createUI(final Composite parent) {
             GridDataFactory.fillDefaults().align(SWT.FILL, SWT.CENTER).applyTo(label);
 
             _txtName = new Text(container, SWT.BORDER);
-            GridDataFactory.fillDefaults().grab(true, false).applyTo(_txtName);
+            GridDataFactory.fillDefaults().span(3, 1).grab(true, false).applyTo(_txtName);
          }
          {
-            // Text: Notes
+            // Text: Image File Path
+            final Label label = UI.createLabel(container, UI.EMPTY_STRING);
+            label.setText(Messages.Dialog_TourTag_Label_Image);
+            GridDataFactory.fillDefaults().align(SWT.FILL, SWT.BEGINNING).applyTo(label);
 
+            _canvasTagImage = new Label(container, SWT.TRANSPARENCY_ALPHA);
```

Comment:
>I see the tag image only in the tag dialog and in the context menu. I would like to see them also in the Tagged Tours view

I did that originally, but the image displayed in the cell is so small that I don't see how useful it will be for users except it they buy a magnifying glass 😄 

![image](https://user-images.githubusercontent.com/25235665/218593237-d1b8aa6e-fb70-4826-92ec-8bf41c6ea074.png)

>The tag image filepath is not visible anywhere, I would like to see them in the tag dialog (maybe in a tooltip) and in the tag view in the pref dialog
I can add it to the tooltip

>It seems that the tag images are slowing down the loading of the tag context menu, this may can be optimized by caching the tag images.

Can you show me an example of what you mean by "caching the image" ?

Reply:
> > I see the tag image only in the tag dialog and in the context menu. I would like to see them also in the Tagged Tours view
> 
> I did that originally, but the image displayed in the cell is so small that I don't see how useful it will be for users except it they buy a magnifying glass 😄

I bought a magnifying glass a few weeks ago 😄 

I would like to see at least in the Tagging view in the pref dialog which tags are containing tag images. It would be helpful to have a new column which displays the image filepathname

Reply:
> > It seems that the tag images are slowing down the loading of the tag context menu, this may can be optimized by caching the tag images.
> 
> Can you show me an example of what you mean by "caching the image" ?

### Simple
net.tourbook.preferences.GraphColorPainter._imageCache

### Complex
de.byteholder.geoclipse.map.TileImageCache
net.tourbook.photo.PhotoImageCache

Reply:
ImageCache implemented. See [here](https://github.com/FJBDev/mytourbook/blob/7b762688222bc2fd35a00ce3d58aa689feee1819/bundles/net.tourbook/src/net/tourbook/tag/TagMenuManager.java#L101).

Reply:
>I would like to see at least in the Tagging view in the pref dialog which tags are containing tag images. It would be helpful to have a new column which displays the image filepathname

Implemented in the "Tour Tags" and "Tagged Tours" views

![image](https://user-images.githubusercontent.com/25235665/219702390-4309d713-ad60-4d8a-8421-fa601a35cab7.png)

Reply:
Can you add a image file path name column also in the tagging pref dialog because there you define the tags and would be helpful for me

![image](https://user-images.githubusercontent.com/1283445/219848685-3609f46f-7c62-4287-b5a3-cb6b374d883e.png)

Reply:
Sure, I will add it

Reply:
Column added

![image](https://user-images.githubusercontent.com/25235665/220470892-bc93fb45-7a28-48fe-a06b-93d369fddde7.png)